results-processor/wptreport.py

```diff
@@ -523,20 +543,22 @@ def prepare_labels(report: WPTReport,
     Returns:
         A set of strings.
     """
+    browser = report.run_info['product']
+    # browser_channel is an optional field.
+    channel = report.run_info.get('browser_channel')
```

Comment:
Instead of the comment, could use `get('browser_channel', None)`, which shows it may not exist and what we want in that situation.

Reply:
That's the same as `get(...)` (`get` returns the value, or None if not found). The comment was meant to point out why the two fields are looked up differently.

Reply:
Right, I was suggesting it as a way of documenting it via-code. But maybe future-us would just wander along and simplify it back to `get('browser_channel')` anyway :). Consider the suggestion very-optional-and-maybe-wrong :D